pkg/provider/ibmsecurityverify/api_client_test.go

```diff
@@ -0,0 +1,116 @@
+package ibmsecurityverify
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/redhat-cop/group-sync-operator/pkg/provider/ibmsecurityverify"
```

Comment:
This is not needed and causes a `import cycle not allowed in` error

Reply:
@sabre1041 Strangely, my build was passing with this line present, but I see VSCode did flag it. When I remove this and update the `ApiClient` imports, the unit tests can no longer be run:

```
go test pkg/provider/ibmsecurityverify/api_client_test.go
# command-line-arguments [command-line-arguments.test]
pkg/provider/ibmsecurityverify/api_client_test.go:52:12: undefined: ApiClient
pkg/provider/ibmsecurityverify/api_client_test.go:76:12: undefined: ApiClient
pkg/provider/ibmsecurityverify/api_client_test.go:107:12: undefined: ApiClient
FAIL    command-line-arguments [build failed]
FAIL
```
Trying alternative ways to run the tests also fails:

```
go test -run TestGetGroup
?       github.com/redhat-cop/group-sync-operator       [no test files]
```

Any ideas what's going wrong here?

Reply:
Nevermind the above, I realized the correct command to run all tests is `go test ./...`, or just to run my own new tests is `go test github.com/redhat-cop/group-sync-operator/pkg/provider/ibmsecurityverify`.

Reply:
was going to suggest changing the syntax, but looks like you were able to make progress